package.json

```diff
@@ -59,8 +59,8 @@
     ]
   },
   "devDependencies": {
-    "@aws-amplify/core": "latest",
-    "@aws-amplify/geo": "latest",
+    "@aws-amplify/geo": "3.0.1-console-preview.4b7d177.0",
```

Comment:
Can we start a list of updates we need to make for launch? We'll need to update this.

Reply:
added in asana and in quip